frontend/index.js

```diff
@@ -1,34 +1,23 @@
 "use strict"
 
-var wasm
+let wasmModule, wasmInst
+let sourcePtr, sourceLength
+const runButton = document.getElementById("run")
```

Comment:
How come you make this a global? It's only used in `initWasm()` it looks like.

Reply:
sorry, that was from my follow up work where I changed it to a `Stop....` while running and made it a cancel button as well. Reverted.